Dockerfile

```diff
@@ -52,6 +52,7 @@ COPY --from=builder /dependencies /usr/local
 ENV PYTHONPATH=/usr/local
 
 EXPOSE 5000
+STOPSIGNAL SIGINT
```

Comment:
Please add a comment above this as to why we have this

```
# this is so that ....
```

Reply:
yeah hmm better off to fix the problem directly then perhaps

here is the test https://github.com/dgtlmoon/changedetection.io/pull/1853/files

without a passing test we cant merge it in

Reply:
> But I think this test will fail, because you are manually sending SIGTERM while this pullrequest doesnt fix the handling of that signal. It only changes the Dockerfile so that the Docker daemon always sends SIGINT instead.

@kroese yeah absolutely, that is true, so it's best to make a PR that fixes the root cause of the issue I think